AUTHORS

```diff
@@ -181,7 +181,6 @@ Ryan Petrello <lists@ryanpetrello.com>
 Fuji Goro <gfuji@cpan.org>
 Siddharth Mahendraker <siddharth_mahen@hotmail.com>
 Dave Pacheco <dap@joyent.com>
-Mathias Buus <m@ge.tt>
```

Comment:
@jasnell some people who have had duplicates there (including Mathias Buus, btw) expressed their opinion in https://github.com/nodejs/node/issues/6403#issuecomment-215053657.